cli/docker/docker.go

```diff
@@ -137,7 +138,14 @@ func (handler *Handler) StartContainer(ctx context.Context, c config.JobConfigur
 		portBindings map[nat.Port][]nat.PortBinding
 	)
 
-	ports, portBindings, err := nat.ParsePortSpecs([]string{"9222:9222"})
+	port, err := freeport.GetFreePort()
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
@christian-bromann can we add a small comment on the reason why we're creating this port binding here, so people can easily understand its purpose. 
Something like:
// binding port for accessing chromeDevTools